src/lib/store-test.js

```diff
@@ -233,4 +233,19 @@ describe("lib/store", () => {
       expect(result()).to.deep.equal(["4", "0", "2", "3", "1"]);
     });
   });
+
+  describe("colors backgrounds", () => {
+    it("should set advance colors", () => {
```

Comment:
advance -> advanced.

But this would result in redundancy after following my previous suggestion of renaming the `description` label. Let's change it to `should set ntp_background`